server/subscriptions.go

```diff
@@ -98,6 +100,28 @@ func (p *Plugin) Subscribe(ctx context.Context, githubClient *github.Client, use
 	return nil
 }
 
+func (p *Plugin) GetSubscriptionsByChannel(channelID string) ([]*Subscription, error) {
+	var filteredSubs []*Subscription
+	subs, err := p.GetSubscriptions()
+	if err != nil {
+		return nil, err
+	}
+
+	for repo, v := range subs.Repositories {
+		for _, s := range v {
+			if s.ChannelID == channelID {
+				// this is needed to be backwards compatible
```

Comment:
0/5 on whether this matters, but it might make sense to do this when we load the subscriptions from the key value store (and when we add new ones, etc.), vs. special casing it here. It may not matter in the short term.

Reply:
I would agree but I think this is good enough for now